server/src/main/resources/org/cloudfoundry/identity/uaa/db/mysql/V4_110__Add_ExternalKey_To_Identity_Provider.sql

```diff
@@ -0,0 +1,3 @@
+-- add column external_key for oauth2,oidc,saml2 IdPs
+ALTER TABLE identity_provider ADD COLUMN external_key TEXT DEFAULT NULL;
+CREATE UNIQUE INDEX external_key_in_zone on identity_provider (identity_zone_id(36),type(8),external_key(724));
```

Comment:
you see, mysql cannot use the full text index but I had to limit it....
Will think about to change to VARCHAR(512) maybe...

Reply:
changed